### DISCUSSION TITLE
HifiGAN Vocoder for custom fine-tuned tacatron2 DDC model

### DISCUSSION THREAD

**Discussion by buczekEngineering** [coqui-ai/TTS#1520] [General]
I'm finetuning the thorsten german pre-trained model with male audio samples. I use the same configurations for finetuning as they were used for initial training. 
I use 1h our of audio. 1 300 audio samples. Phoneme based training. Coqui TTS v.0.5.0
After 20k-30k steps the model output (using the default vocoder) sounds similar but still many artefacts are audible, the quality is in generall very low. 
I would be thankful if someone would say how can I improve it? 
- longer fine-tuning? 
- changing the r param? [ I finetuned with r=2, because of the gradual training it was reduced to 2. Is it okay?]
- training own HifiGan Vocoder?  [is it important to train/finetune vocoder for each voice? Would it improve the quality of the generated voice?]

**Comment by erogol** [comment]
There are some HifiGAN fixes upcoming to the dev branch. I'll keep you posted when it is landed.
